ReflectometryServer/ChannelAccess/pv_manager.py

```diff
@@ -246,27 +252,37 @@ def _add_all_parameter_pvs(self, param_types):
                 keyed by name.
         """
         for param, (param_type, group_names, description) in param_types.items():
-            self._add_parameter_pvs(param, group_names, description, **PARAMS_FIELDS_BEAMLINE_TYPES[param_type])
+            self._add_parameter_pvs(param, group_names, description, param_type)
         self.PVDB[TRACKING_AXES] = {'type': 'char',
                                     'count': 300,
                                     'value': json.dumps(self._tracking_positions)
                                     }
-
-    def _add_parameter_pvs(self, param_name, group_names, description, **fields):
+    
+    def _add_parameter_pvs(self, param_name, group_names, description, param_type):
         """
         Adds all PVs needed for one beamline parameter to the PV database.
 
         Args:
             param_name: The name of the beamline parameter
-            fields: The fields of the parameter PV
-            group_names: list fo groups to which this parameter belong
+            param_type: The type of the parameter
+            group_names: list of groups to which this parameter belong
             description: description of the pv
         """
         try:
             param_alias = create_pv_name(param_name, self.PVDB.keys(), "PARAM", limit=10)
             prepended_alias = "{}:{}".format(PARAM_PREFIX, param_alias)
-            if BeamlineParameterGroup.TRACKING in group_names:
-                self._tracking_positions[prepended_alias] = param_name
+           
+            fields = PARAMS_FIELDS_BEAMLINE_TYPES[param_type]
+            # generate a dictionary to store metadata about tracking positions
+            item = {}
```

Comment:
`item` is not very descriptive - perhaps `param_info` instead?

Reply:
renamed in latest commit